CMP1903_A1_2324/Die.cs

```diff
@@ -1,23 +1,22 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace CMP1903_A1_2324
 {
     internal class Die
     {
-        /*
-         * The Die class should contain one property to hold the current die value,
-         * and one method that rolls the die, returns and integer and takes no parameters.
-         */
-
-        //Property
-
-
-        //Method
+        private int _dieValue; //creates a private variable for _dieValue, so that it couldn't be used by other c# files linked
```

Comment:
Good code to specify the variables / objects, code is self explanatory, even without comments.